bears/stylus/StylintBear.py

```diff
@@ -0,0 +1,206 @@
+                        allow_zero_units: str='never',
+                        z_index_normalize: bool=False):
+        """
+        :param allow_block_keyword:
+            When ``always`` expect the ``@block`` keyword when defining block
+            variables. When ``never``, expect no ``@block`` keyword when
+            defining block variables. When ``false``, do not check either way.
+        :param allow_brackets:
+            When ``always``, expect ``{}`` when declaring a selector. When
+            ``never``, expect no brackets when declaring a selector.
+        :param allow_colons:
+            When ``always``, expect ``:`` when declaring a property. When
+            ``never``, expect no ``:`` when declaring a property.
+        :param allow_colors:
+            When ``always``, enforce variables when defining hex values.
+        :param allow_spaces_after_commas:
+            Enforce or disallow spaces after commas.
+        :param allow_spaces_after_comments:
+            Enforce or disallow spaces after line comments.
+        :param allow_css_literal:
+            By default Stylint ignores ``@css`` blocks. If set to true however,
+            it will throw a warning if ``@css`` is used.
+        :param max_selector_depth:
+            Set the max selector depth. If set to 4, max selector depth will
+            be 4 indents. Pseudo selectors like ``&:first-child`` or
+            ``&:hover`` won't count towards the limit.
+        :param check_duplicates:
+            Checks if selector or property duplicated unnecessarily. By
+            default, only checks on a file-by-file basis, but if
+            ``globalDupes: true`` is set, then it will also check for
+            duplicates globally (for root values only).
+        :param allow_efficient_properties:
+            Check for places where properties can be written more efficiently.
+        :param set_extend_preference:
+            Pass in either ``@extend`` or ``@extends`` and then enforce that.
+            Both are valid in Stylus. It doesn't really matter which one
+            you use.
+        :param check_global_duplicates:
+            Works in conjunction with duplicates. Does nothing on its own.
+            If ``false``, duplicates will check for dupes within individual
+            files only. If true, duplicates will check for dupes across all
+            files.
+        :param set_indent_preference:
+            This works in conjunction with ``max_selector_depth``. If you
+            indent with spaces this is the number of spaces you indent with.
+            If you use hard tabs, set this value to ``false``.
+        :param allow_leading_zero:
+            Enforce or disallow unnecessary leading zeroes on decimal points.
+        :param set_max_errors:
+            Set 'max' number of Errors.
+        :param set_max_warnings:
+            Set 'max' number of Warnings.
+        :param mixed_spaces_and_tabs:
```

Comment:
Find a way to hide their config variables from our users.
Our users want to use the standard config variables that are the same across all linters.
They do not want to understand the complexities of each linter's config.